src/js/components/DataTable/stories/ColumnSize.js

```diff
@@ -0,0 +1,196 @@
+  { property: 'date', header: 'Date', size: '1/4' },
+];
+
+const columnsAbsoluteSize = [
+  { property: 'name', header: 'Name', size: '600px' },
+  { property: 'location', header: 'Location', size: '200px' },
+  { property: 'date', header: 'Date', size: '200px', align: 'end' },
+  { property: 'percent', header: 'Percent', size: '100px', align: 'end' },
+  { property: 'paid', header: 'Paid', size: '100px', align: 'end' },
+];
+
+const columnsDefault = [
+  { property: 'name', header: 'Name' },
+  { property: 'location', header: 'Location' },
+  { property: 'date', header: 'Date', align: 'end' },
+  { property: 'percent', header: 'Percent', align: 'end' },
+  { property: 'paid', header: 'Paid', align: 'end' },
+];
+
+const Example = () => (
+  <Grommet theme={grommet}>
+    <Box fill="horizontal" pad="medium">
+      <Heading level="3"> Default DataTable</Heading>
+      <DataTable
+        columns={columnsDefault}
+        data={DATA}
+        step={10}
+        primaryKey={false}
+        border={{
+          color: 'border',
+          side: 'vertical',
+          size: '1px',
+        }}
+      />
+    </Box>
+
+    <Box fill="horizontal" pad="medium">
+      <Heading level="3">Theme Column Sizes</Heading>
+      <DataTable
+        columns={columnsThemeSize}
+        data={DATA}
+        step={10}
+        primaryKey={false}
+        border={{
+          color: 'border',
+          side: 'vertical',
+          size: '1px',
+        }}
+      />
+    </Box>
+
+    <Box fill="horizontal" pad="medium">
+      <Heading level="3">Absolute Column Sizes</Heading>
+      <DataTable
+        columns={columnsAbsoluteSize}
+        data={DATA}
+        step={10}
+        primaryKey={false}
+        border={{
+          color: 'border',
+          side: 'vertical',
+          size: '1px',
+        }}
+      />
+    </Box>
+
+    <Box fill="horizontal" pad="medium">
+      <Heading level="3">Relative Column Sizes</Heading>
+      <DataTable
+        columns={columnsRelativeSize}
+        data={DATA}
+        step={10}
+        primaryKey={false}
+        border={{
+          color: 'border',
+          side: 'vertical',
+          size: '1px',
+        }}
+      />
+    </Box>
+  </Grommet>
+);
+
+const ExampleResizable = () => (
+  <Grommet theme={grommet}>
+    <Box align="center" pad="medium">
+      <Heading level="3">Large table with resizeable</Heading>
+      <DataTable
+        columns={columnsThemeSize}
+        data={DATA}
+        step={10}
+        primaryKey={false}
+        resizeable
+      />
+    </Box>
+
+    <Box align="center" pad="medium">
+      <Heading level="3">Small table with resizeable</Heading>
+      <DataTable
+        columns={columnsReize}
+        data={DATA}
+        step={10}
+        primaryKey={false}
+        resizeable
+      />
+    </Box>
+  </Grommet>
+);
+storiesOf('DataTable', module).add('Column Sizes', () => <Example />);
+
+storiesOf('DataTable', module).add('Column Sizes resizeable', () => (
```

Comment:
From user feedback, we found that separating the files of the stories is more readable and easier to navigate inside the code. 
Could you please separate this file into two files for each story? you can share the common data with importing it from one file to another and add a comment with link of where this data is taken from. It will help the users navigate.

Reply:
Done!